### DISCUSSION TITLE
Issues with map provider

### DISCUSSION THREAD

**Discussion by ovahead** [mytourbook/mytourbook#1062] [General]
Everytime I start MT the 2D Tour Map view starts with SRTM as map provider after changing it once to SRTM. If I change the map provider back to OSM and click apply and close, next time MT starts again with SRTM as map provider :(

I know that OSM had an green circle before I tried to change it to SRTM but now both provider had always a red circle before the name of the provider.

![Map-View-and-Map-Provider](https://user-images.githubusercontent.com/112829463/231763004-90d42894-4ef9-41c3-abed-f43eba395175.jpg)

**Comment by wolfgang-ch** [comment]
A map provider is selected in this slideout and this selection is preserved until another map provider is selected in this slideout, also after restarting MT.

![image](https://user-images.githubusercontent.com/1283445/231769046-941dbb5f-65bd-4e8a-ade1-c142414f51f2.png)

**Reply by ovahead** [reply]
Hm, okay. Thanks very much!
So the "apply and close" button in map provider is useless to choose a provider?

**Reply by wolfgang-ch** [reply]
> So the "apply and close" button in map provider is useless to choose a provider?

It is not useless, it is used to save a modified map provider

The map provider preference is used to setup the map providers (add/delete/configure) and the map slideout is used to select a map provider.

It may be irritating that a selected map provider in the pref page is also displayed immediately in the map view but this is helpful when a map provider is setup.
